google-cloud-speech/lib/google/cloud/speech/service.rb

```diff
@@ -29,15 +29,16 @@ class Service
 
         ##
         # Creates a new Service instance.
-        def initialize project, credentials, timeout: nil, client_config: nil
+        def initialize project, credentials, timeout: nil, client_config: {}
```

Comment:
The problem with this change is what happens if `nil` is passed in. The GRPC code expects a hash, and this change could cause errors. @quartzmo what do you think?

Reply:
@blowmage I agree.

@IlyaUmanets What is your motivation for the changes in this PR? They appear to be stylistic refactorings, but I am concerned that despite the project's automated testing, there is a very slight risk of introducing functional changes that I do not see justified by any benefit, unless you can propose one.